README.md

```diff
@@ -215,7 +226,35 @@ vulnerabilities as well as configuration issues that might affect stability, rel
 To learn more about the available Starboard commands and scanners, such as [kube-bench][aqua-kube-bench] or
 [kube-hunter][aqua-kube-hunter], use `starboard help`.
 
-## Configuration
+## Custom Security Resources Definitions
+
+This project houses CustomResourceDefinitions (CRDs) related to security and compliance checks along with the code
+generated by Kubernetes [code generators][k8s-code-generator] to write such custom resources in a natural way.
+
+| NAME                                             | SHORTNAMES   | APIGROUP               | NAMESPACED |  KIND               |
+| ------------------------------------------------ | ------------ | ---------------------- | ---------- | ------------------- |
+| [vulnerabilityreports][vulnerabilityreports-crd] | vulns,vuln   | aquasecurity.github.io | true       | VulnerabilityReport |
+| [configauditreports][configauditreports-crd]     | configaudit  | aquasecurity.github.io | true       | ConfigAuditReport   |
+| [ciskubebenchreports][ciskubebenchreports-crd]   | kubebench    | aquasecurity.github.io | false      | CISKubeBenchReport  |
+| [kubehunterreports][kubehunterreports-crd]       | kubehunter   | aquasecurity.github.io | false      | KubeHunterReport    |
+
+See [Custom Security Resources Specification][starboard-crds-spec] for the detailed explanation of custom resources
+used by Starboard and their lifecycle.
+
+## Starboard CLI
+
+Starboard CLI is a single executable binary which can be used to find risks, such as vulnerabilities or insecure Pod
+specs, in Kubernetes workloads. By default, the risk assessment reports are stored as
+[custom security resources][starboard-crds].
+
+To learn more about the available Starboard CLI commands, run `starboard help` or type a command followed by the
+`-h` flag:
+
+```
+$ starboard kube-hunter -h
+```
+
+### Configuration
```

Comment:
Am I right to think that the Starboard operator will create the vulnerabilityreports CRD - no need to run `starboard init` first?

Reply:
It depends how we install the operator. With OLM the OLM operator is taking care of creating CRDs, which are part of the installation bundle. For helm or kubecly apply installations we have to create the CRDs manually or use the init command.

In general, as a good practice the operator should not create the CRDs that it manages.

Reply:
Helm will install them if they dont already excist now, but not as the other resources. It will also not try to update them or similarly if, just warn about it if needed.

Reply:
This is great that a Helm chart package now supports CRDs that are installed before a Helm release is created. Even though it's not as powerful and automated as OLM, it's useful in case of Helm chart installation option